evmcc/evmcc.go

```diff
@@ -127,7 +127,8 @@ func (evmcc *EvmChaincode) Invoke(stub shim.ChaincodeStubInterface) pb.Response
 		}
 
 		// Passing the first 8 bytes contract address just created
-		err := eventSink.Flush(string(contractAddr.Bytes()[0:8]))
+		// Hex Encode before flushing to ensure no non utf-8 characters
+		err := eventSink.Flush(hex.EncodeToString(contractAddr.Bytes()[0:8]))
```

Comment:
The bytes range should be [0:4] because one byte will be encoded as two byte hex string.

Reply:
@swetharepakula resolve or dismiss, please.